### PR TITLE
Added simple JSX testing files

### PR DESCRIPTION
**Pull request by michbil**

### CODE REVIEW

**Comment by AlexeyAnshakov** [comment]
@spirinvladimir , please review

File: test/test.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        },
+        waitSeconds : 0,
+        baseUrl: "/",
+
+        paths: {
+            "jquery": "https://code.jquery.com/jquery-2.1.3.min",
+            "bootstrap":"https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min",
+            "react": "https://fb.me/react-0.13.3",
+            "JSXTransformer": "JSXTransformer",
+            "login":"login",
```
Comment (spirinvladimir) [inline test/test.html]: Am I right?
```
"login":"widget/login",
```

Comment (michbil) [inline test/test.html]: No, it hosted at /login.jsx

**Comment by AlexeyAnshakov** [comment]
@spirinvladimir is it ready?

**Pull request merged**